.github/workflows/docs.yml

```diff
@@ -0,0 +1,35 @@
+name: Docs
+
+on:
+  push:
+    branches: [main]
+
+jobs: 
+  lint: 
+    name: Lint
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+        with:
+          ref: main
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install Moonwave
+        run: npm i -g moonwave
```

Comment:
This might fail (I'm not sure, but you may need root permissions to install a package globally on Ubuntu). Consider installing moonwave locally (without `-g`) and invoking it with `./node_modules/.bin/moonwave`.

Reply:
I believe the workflow should work, as actions do give root permissions.